sdk/storageactions/azure-resourcemanager-storageactions/CHANGELOG.md

```diff
@@ -1,14 +1,61 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
+## 1.0.0 (2025-06-30)
 
-### Features Added
+- Azure Resource Manager StorageActions client library for Java. This package contains Microsoft Azure SDK for StorageActions Management SDK. The Azure Storage Actions Management API. Package api-version 2023-01-01. For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
 
 ### Breaking Changes
 
-### Bugs Fixed
+#### `models.StorageTaskReportSummary` was removed
+
+#### `models.StorageTasksListResult` was removed
+
+#### `models.StorageTaskAssignmentsListResult` was removed
+
+#### `models.OperationListResult` was removed
+
+#### `models.StorageTask$Update` was modified
+
+* `withProperties(models.StorageTaskProperties)` was removed
+
+#### `models.StorageTaskUpdateParameters` was modified
+
+* `models.StorageTaskProperties properties()` -> `models.StorageTaskUpdateProperties properties()`
+* `withProperties(models.StorageTaskProperties)` was removed
```

Comment:
This one due to migration to TypeSpec, the payload for update is now `StorageTaskUpdateProperties`